Source/Core/Core/BootManager.h

```diff
@@ -6,9 +6,11 @@
 
 #include <string>
 
+#include "Core/ConfigManager.h"
+
 namespace BootManager
 {
-bool BootCore(const std::string& _rFilename);
```

Comment:
Is a default argument here *really* necessary? There's only three occurrences where specifying the boot type would be necessary with `BOOT_DEFAULT`.